webext/src/dom/manager.js

```diff
@@ -210,15 +212,32 @@ export default class extends utils.mixins(CommonMixin, CommandsMixin) {
     let target = document.querySelector("body");
     let observer = new MutationObserver(mutations => {
       mutations.forEach(mutation => {
+        if (!target) {
```

Comment:
Interesting! Can you tell me more about this bit?

Reply:
I see type error from logs, makes me believe the body node some times doesn't exist at that time.